posydon/interpolation/profile_interpolation.py

```diff
@@ -134,9 +137,14 @@ def scrape(self,grid,ind,hms_s2):
         
         if 'omega' in self.names:
             profiles[-1]= profiles[self.names.index('omega')]/  \
-                          grid.final_values['S1_surf_avg_omega_div_omega_crit'][ind]
+                          (grid.final_values['S1_surf_avg_omega'][ind]/ \
+                           grid.final_values['S1_surf_avg_omega_div_omega_crit'][ind])
         
-        return scalars, profiles
+        if not np.isnan(grid.final_values['S1_surf_avg_omega_div_omega_crit'][ind]):
```

Comment:
When I remember correctly, at some point we said to not use `np.isnan` but `pd.isna` instead. See issue #238 , thus I'd encourage to not add new `np.isnan`s to the code.

Reply:
fixed, thanks!